quote.py

```diff
@@ -3,48 +3,102 @@
 import random
 from halibot import HalModule
 
+sr = random.SystemRandom()
```

Comment:
Is this needed? Docs say this isn't necessarily implemented on all platforms, which gets a NACK from me.